e2e/utils/runtime-utils.ts

```diff
@@ -19,12 +20,8 @@ export async function initializeRuntimeIfModalPresented(page: Page): Promise<voi
 }
 
 async function isSecuritySuspended(page: Page): Promise<boolean> {
-  try {
-    await page.waitForXPath('//*[@data-test-id="security-suspended-msg"]', { visible: true });
-  } catch (e) {
-    return false;
-  }
-  return true;
+  const xpath = '//*[@data-test-id="security-suspended-msg"]';
```

Comment:
nit: I don't think this separate variable is improving readability, would just inline

Reply:
Done